deploy/terraform/main.tf

```diff
@@ -21,75 +22,72 @@ data "aws_ssm_parameter" "ami" {
 ##################################################################################
 
 # NETWORKING #
-resource "aws_vpc" "vpc" {
-  cidr_block           = "10.0.0.0/16"
+resource "aws_vpc" "opensource-vpc" {
+  cidr_block           = "172.31.0.0/16"
   enable_dns_hostnames = true
-
 }
 
 resource "aws_internet_gateway" "igw" {
-  vpc_id = aws_vpc.vpc.id
-
+  vpc_id = aws_vpc.opensource-vpc.id
 }
 
-resource "aws_subnet" "subnet1" {
-  cidr_block              = "10.0.0.0/24"
-  vpc_id                  = aws_vpc.vpc.id
+resource "aws_subnet" "us-east-1b" {
+  cidr_block              = "172.31.80.0/20"
+  vpc_id                  = aws_vpc.opensource-vpc.id
   map_public_ip_on_launch = true
 }
 
 # ROUTING #
 resource "aws_route_table" "rtb" {
-  vpc_id = aws_vpc.vpc.id
+  vpc_id = aws_vpc.opensource-vpc.id
 
   route {
-    cidr_block = "0.0.0.0/0"
+    cidr_block = "172.31.0.0/16"
     gateway_id = aws_internet_gateway.igw.id
   }
 }
 
 resource "aws_route_table_association" "rta-subnet1" {
-  subnet_id      = aws_subnet.subnet1.id
+  subnet_id      = aws_subnet.us-east-1b.id
   route_table_id = aws_route_table.rtb.id
 }
 
+resource "aws_ecs_service" "osmt_container" {
+  name = "osmt_app"
+}
+
+resource "aws_db_instance" "osmt_db" {
+  allocated_storage    = 10
+  db_name              = "osmt_db"
+  engine               = "mysql"
+  engine_version       = "5.7"
+  instance_class       = "db.t3.micro"
+  username             = "osmt_user"
+  password             = "password"
```

Comment:
Should this be a secret?

Reply:
yes. I have removed the string and added it as and ENV variable